pkgs/development/libraries/rutabaga_gfx/default.nix

```diff
@@ -29,15 +30,27 @@ stdenv.mkDerivation (finalAttrs: {
       decode = "base64 -d";
       hash = "sha256-Ji1bK7jnRlg0OpDfCLcTHfPSiz3zYcdgsWL4n3EoIYI=";
     })
+    # Fix error in Makefile where it uses eight spaces instead of a tab
+    # https://chromium-review.googlesource.com/c/crosvm/crosvm/+/4863380
+    (fetchpatch {
+      url = "https://chromium.googlesource.com/crosvm/crosvm/+/fc415bccc43d36f63a2fd4c28878591bb1053450%5E%21/?format=TEXT";
+      decode = "base64 -d";
+      hash = "sha256-SLzlZ4o1+R2bGTPvA0a5emq97hOIIIHrubFhcQjqYwg=";
+    })
+    # Install the dylib on Darwin.
+    ./darwin-install.patch
   ];
 
   nativeBuildInputs = [ cargo pkg-config rustPlatform.cargoSetupHook ];
-  buildInputs = lib.optionals (lib.meta.availableOn stdenv.hostPlatform gfxstream) ([
-    aemu
-    gfxstream
-  ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform libdrm) [
-    libdrm
-  ]);
+  buildInputs =
+    lib.optionals stdenv.isDarwin [ libiconv ]
+    ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform gfxstream) ([
+      aemu
+      gfxstream
+    ]
+    ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform libdrm) [
```

Comment:
Does this need to be reformatted?

Reply:
The whole file or just that block?

Reply:
Well, any of it?

Reply:
I guess that was kind of an odd question. I’ve been reformatting packages I maintain (or substantially change) with the RFC-166 formatter (nixpkgs-rfc-style) but holding off for other things. I went ahead and reformatted both. The force-push should have the changes.

Reply:
Ah, no, sorry — I meant, why did any of the formatting need to be changed, from the perspective that I'd rather have had the diff as small as possible than having any of the formatting changed, since formatting discussions are still ongoing.

Reply:
Ah, okay. I’ll revert the formatting change and try to reduce the size further.

Reply:
Done. I re-pushed with the formatting back the way it was and the formatting change to the `buildInputs` in rutabaga_gfx also reverted.